openai/openai.go

```diff
@@ -244,6 +244,20 @@ func toChatCompletion(id string, r api.ChatResponse) ChatCompletion {
 }
 
 func toChunk(id string, r api.ChatResponse) ChatCompletionChunk {
+	toolCalls := make([]ToolCall, len(r.Message.ToolCalls))
```

Comment:
This logic is the same as in `toChatCompletion` - just pulled out for now even though theoretically there is only one tool call per chunk from routes logic